cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -178,24 +178,23 @@ public void handleLine(String line) {
         annotateRecordsWithPOST(recordsToAnnotate);
     }
 
+    // doesn't seem like this method needs to return anything, could be a void function
```

Comment:
let's just remove this comment then and make the method return void instead of leaving it as is.

Reply:
already done